core/scripts/dkg/setup.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/smartcontractkit/chainlink/core/cmd"
+	"github.com/smartcontractkit/chainlink/core/config"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/recovery"
+	helpers "github.com/smartcontractkit/chainlink/core/scripts/common"
+	dkgContract "github.com/smartcontractkit/ocr2vrf/gethwrappers/dkg"
+	"github.com/urfave/cli"
+)
+
+func main() {
+	recovery.ReportPanics(func() {
+		Run(NewProductionClient(), os.Args...)
+	})
+}
+
+func Run(client *cmd.Client, args ...string) {
+
+	app := cmd.NewApp(client)
+	e := helpers.SetupEnv()
+
+	_, tx, _, err := dkgContract.DeployDKG(e.Owner, e.Ec)
+	helpers.PanicErr(err)
+	dkgAddress := helpers.ConfirmContractDeployed(context.Background(), e.Ec, tx, e.ChainID).String()
+
+	onChainPublicKeys := []string{}
+	offChainPublicKeys := []string{}
+	configPublicKeys := []string{}
+	peerIDs := []string{}
+	transmitters := []string{}
+	dkgEncrypters := []string{}
+	dkgSigners := []string{}
+
+	for i := 0; i < 5; i++ {
```

Comment:
Perhaps this `5` here should be configurable (but can ensure at least 5 is provided to run DKG).